### PR TITLE
docs(deps): update list of peer dependencies #129

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #129

## Summary:

Update documented list of peer dependencies

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Docs

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by blackfalcon** [comment]
You missed the update with the CDN bundles. The reference to design-tokens is incorrect. 

```
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/bundled/essentials.css" />
<script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-datepicker@latest/dist/auro-datepicker__bundled.js" type="module"></script>
```

Also, see the screenshot. When using the CDN bundles, it's not appearing as the icons are coming through. 

Screen Shot 2023-06-13 at 10 41 19 AM

File: README.md [hunk 1]
```diff
@@ -43,10 +43,7 @@ Installing as a direct, dev or peer dependency is up to the user installing the
 The following Auro peer dependencies must be included.
 
 ```shell
-$ npm i @aurodesignsystem/auro-icon
-$ npm i @aurodesignsystem/auro-calendar
-$ npm i @aurodesignsystem/auro-dropdown
-$ npm i @aurodesignsystem/auro-input
+$ npm i alaskaairux/design-tokens
```
Comment (blackfalcon) [inline README.md:46]: Please update the CDN reference content too.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by jason-capsule42** [comment]
> You missed the update with the CDN bundles. The reference to design-tokens is incorrect.
> 
> ```
> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/bundled/essentials.css" />
> <script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-datepicker@latest/dist/auro-datepicker__bundled.js" type="module"></script>
> ```
> 
> Also, see the screenshot. When using the CDN bundles, it's not appearing as the icons are coming through.
> 
> Screen Shot 2023-06-13 at 10 41 19 AM

They seem to be working for me?

Screenshot 2023-06-13 at 12 50 47 PM

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**